src/app/campaign.model.ts

```diff
@@ -33,16 +33,16 @@ export interface Campaign {
      * ISO 8601 formatted datetime
      **/
     endDate: string;
-    impactReporting: string;
-    impactSummary: string;
+    impactReporting: string | null;
+    impactSummary: string | null;
     isMatched: boolean;
     matchFundsRemaining: number;
     matchFundsTotal: number;
     parentUsesSharedFunds: boolean;
-    problem: string;
+    problem: string | null;
     quotes: Array<{person: string, quote: string}>;
     ready: boolean;
-    solution: string;
+    solution: string | null;
```

Comment:
I've found three of these properties to be null for the campaign I'm testing with in `full`, only `problem` is not null but that's also not blank. I think it's safest to treat them all as nullable, particularly while they're being sent from apex which doesn't mark nullability in its type system.